js/main.js

```diff
@@ -1,124 +1,344 @@
+  bowlWidth = 90;
 
-let ctx = canvas.getContext("2d");
 canvas.width = 400;
 canvas.height = 600;
 
+//Назначим изначальные координаты появления первого фрукта и миски
+let food = {
+  x: Math.floor(Math.random() * (canvas.width - foodWidth)),
+  y: -15,
+};
+const refr = {
+  x: canvas.width / 2 - bowlWidth / 2,
+  y: canvas.height - (bowlHeight - 10),
+};
+//
+
+//Сборник картинок <-- Блок снизу
 const background = new Image();
-background.src = "img/bg.jpg";
+background.src = 'img/bg.jpg';
 
 const bowl = new Image();
-bowl.src = "img/bowl.png"
+bowl.src = 'img/bowl.png';
 
-let score = 0,
-foodImg = [],
-randCount = Math.floor(Math.random()*5 + 1),
-dir;
+const scoreImg = new Image();
+scoreImg.src = 'img/score.png';
 
-for ( let i = 1; i <= 5; i++) {
-    foodImg[i] = new Image();
-    foodImg[i].src = "img/" + i + ".png";
+for (let i = 1; i <= 5; i++) {
+  const image = new Image();
+  image.src = `img/${i}.png`;
+  foodImg.push(image);
 }
+let randCount = Math.floor(Math.random() * (foodImg.length - 1));
+/////////////////////////////////
 
- let food = {
-     x: Math.floor(Math.random()*275+15),
-     y: -15,
-}
-let refr = {
-    x: 250,
-    y : 540,
+//Аудио сборник <-- Блок снизу
+const press_button_in_diff = new Audio('audio/knopka.mp3');
 
-}
+const sound_of_horn1 = new Audio('audio/zvyk_gorna1.mp3');
+
+const easy_music = new Audio('audio/easy_music.mp3');
+easy_music.setAttribute('loop', 'loop');
+
+const medium_music = new Audio('audio/medium_music.mp3');
+medium_music.setAttribute('loop', 'loop');
 
+const hard_music = new Audio('audio/hard_music.mp3');
+hard_music.setAttribute('loop', 'loop');
 
-document.addEventListener("keydown", direction);
+const catch_fruit = new Audio('audio/catch_fruit.mp3');
+const catch_bonus = new Audio('audio/catch_bonus.mp3');
+
+const game_over = new Audio('audio/game_over.mp3');
+
+//Обьект где храняться все звуки игры
+const sounds_obj = {
+  1: press_button_in_diff,
+  2: sound_of_horn1,
+  3: easy_music,
+  4: medium_music,
+  5: hard_music,
+  6: catch_fruit,
+  7: catch_bonus,
+  8: game_over,
+};
+//////////////////////////////
+
+/* Функция, которая будет указывать
+направление миски при нажатие на стрелки и WASD */
+document.addEventListener('keydown', direction);
 function direction(event) {
-	if(event.keyCode === 37 || event.keyCode === 97 || event.keyCode === 65 )
-		dir = "left";
-	else if(event.keyCode === 39 || event.keyCode === 68 || event.keyCode === 100)
-        dir = "right";
-        else if (event.keyCode === 40 || event.keyCode === 115 || event.keyCode === 83 )
-        dir = "down";
+  if (event.keyCode === 37 ||
+      event.keyCode === 97 ||
+      event.keyCode === 65) {
+    dir = 'left';
+  } else if (event.keyCode === 39 ||
+           event.keyCode === 68 ||
+           event.keyCode === 100) {
+    dir = 'right';
+  } else if (event.keyCode === 40 ||
+           event.keyCode === 115 ||
+           event.keyCode === 83) {
+    dir = 'down';
+  }
```

Comment:
Use named constants for key codes
Decompose long conditions (no multi-line if-s)